docs/install_on_kestrel.md

```diff
@@ -0,0 +1,217 @@
+
+```
+#!/bin/bash -l
+
+rm -rf build
+mkdir build
+cd build
+
+module purge
+module load craype-x86-spr
+module load intel-oneapi-mpi/2021.10.0-intel
+module load intel-oneapi-compilers/2023.2.0
+module load netcdf-c/4.9.2-intel-oneapi-mpi-intel
+module load git/2.40.0
+module use /nopt/nrel/apps/modules/test/application
+module load helics/3.4.0-cray-mpich-intel
+
+
+module load cmake
+
+cmake -DAMR_WIND_ENABLE_CUDA:BOOL=OFF \
+      -DAMR_WIND_ENABLE_MPI:BOOL=ON \
+      -DAMR_WIND_ENABLE_OPENMP:BOOL=OFF \
+      -DAMR_WIND_TEST_WITH_FCOMPARE:BOOL=OFF \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DAMR_WIND_ENABLE_NETCDF:BOOL=ON \
+      -DNETCDF_DIR:PATH=/nopt/nrel/ecom/hpacf/software/2020-07/spack/opt/spack/linux-centos7-skylake_avx512/gcc-8.4.0/netcdf-c-4.7.3-533s5vfhvbbvpgxambbzk66vtlcce2u6  \
+      -DnetCDF_DIR:PATH=/nopt/nrel/ecom/hpacf/software/2020-07/spack/opt/spack/linux-centos7-skylake_avx512/gcc-8.4.0/netcdf-c-4.7.3-533s5vfhvbbvpgxambbzk66vtlcce2u6  \
+      -DAMR_WIND_ENABLE_OPENFAST:BOOL=ON \
+      -DOpenFAST_ROOT:PATH=/home/pfleming/herc_root/OpenFAST/build/install/ \
+      -DAMR_WIND_ENABLE_HYPRE:BOOL=OFF \
+      -DAMR_WIND_ENABLE_MASA:BOOL=OFF \
+      -DAMR_WIND_ENABLE_HELICS:BOOL=ON \
+      -DAMR_WIND_ENABLE_TESTS:BOOL=ON \
+      -DAMR_WIND_ENABLE_FORTRAN:BOOL=OFF \
+      -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON \
+      -DAMR_WIND_ENABLE_ALL_WARNINGS:BOOL=ON \
+      -DBUILD_SHARED_LIBS:BOOL=ON \
+      ..
+
+nice make -j32
+      -DAMR_WIND_ENABLE_TESTS:BOOL=ON \
+      -DAMR_WIND_ENABLE_FORTRAN:BOOL=OFF \
+      -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON \
+      -DAMR_WIND_ENABLE_ALL_WARNINGS:BOOL=ON \
+      -DBUILD_SHARED_LIBS:BOOL=ON \
+      ..
+
+nice make -j32
+```
+
+Run via:
+
+```
+bash build_script
+```
+
+## Set up a hercules conda environment
+
+This will be needed for the remaining installations
+
+```
+module load anaconda3
+conda create --name hercules python
+conda activate hercules
+```
+
+Edit your ~/.bashrc file to include a helpful shortcut function based on this:
+
+```
+env_hercules()
+{
+        module purge
+        module load craype-x86-spr
+        module load intel-oneapi-mpi/2021.10.0-intel
+        module load intel-oneapi-compilers/2023.2.0
+        module load netcdf-c/4.9.2-intel-oneapi-mpi-intel
+        module load git/2.40.0
+        # module use /nopt/nrel/apps/modules/test/application
+        # module load helics/3.4.0-cray-mpich-intel
+        module load anaconda3
+
+        conda activate hercules
+}
+```
+
+## Install SEAS
+
+Go back to herc_root
+
+```
+git clone https://github.nrel.gov/SEAS/SEAS
+
+cd SEAS
+git fetch --all
+git switch dv/emuwind
+cd ..
+pip install -e SEAS
+```
+
+## Install Hercules
+
+Go back to herc_root
+
+```
+git clone https://github.com/NREL/hercules
+pip install -e hercules
+```
+
+## Install electrolyzer module
+
+Go back to herc_root
+
+```
+git clone git@github.com:NREL/electrolyzer.git
+cd electrolyzer
+git fetch --all
+git switch develop
+```
```

Comment:
Need to add a "pip install -e ." line here to install the electrolyzer

Reply:
thank you!